cmd/nerdctl/login.go

```diff
@@ -205,7 +207,17 @@ func ConfigureAuthentification(authConfig *types.AuthConfig, options *loginOptio
 	}
 
 	if options.username == "" {
-		return fmt.Errorf("error: Username is Required")
+		fmt.Printf("Enter Username:")
```

Comment:
plz add a space suffix and use  `fmt.Print` =>  `fmt.Print("Enter Username: ")`

Reply:
done